srcpkgs/llvm17/template

```diff
@@ -58,8 +59,9 @@ if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 fi
 
 subpackages="libllvm17 llvm17-doc llvm17-devel"
-_enabled_projects=
-_enabled_runtimes=
+
+# use $(:) to silence xlint
+_enabled_projects=$(:)
```

Comment:
https://github.com/leahneukirchen/xtools/pull/323

Reply:
or initialize it with `-DLLVM_ENABLE_PROJECTS=`, but i don't care strongly.